### ISSUE TITLE
Problem fixing cursor to a random message

### ISSUE THREAD

**Issue by ghost** [UnigramDev/Unigram#1183]
Hello all,

I am a blind user of Unigram.

I want to talk about a problem.

My cursor isn't fixed to **unread messages** when I enter a chat from the Chats tab.
The cursor is fixed to another message. It's random. And I don't understand where the conversation starts.  
I usually try to find the beginning of the conversation with the **page down / page up** keys.  
It doesn't work if there are many messages. I can't follow the matter.

My knowledge of c # is very limited. I can't tell you this in the code base.

To summarize, I want my cursor fixed to **unread messages in chats**.

I don't have a strong English. That's why I couldn't tell you.  
You can ask the details, I will try to explain better.

Yours.

**Comment by FrayxRulez** [comment]
From @Neurrone:
> Hi,
> 
> I'm using the NVDA screen reader with Unigram and noticed some bugs with how the shift+tab key works when navigating from the message input box, which automatically gains focus when I press enter on a conversation.
> 
> When the message is scrolled to the bottom (i.e, showing the most recent messages):
> 
> 1. Pressing shift+tab once from message input speaks "list, link". The link is missing a label to indicate what it does.
> 2. Pressing shift+tab a second time speaks "link, " which is fine.
> 3. Pressing shift+tab a third time actually brings the message list item in focus. The problem is that pressing shift+tab from message input when opening a conversation with unread messages often does not bring me to the first unread message, and seems to land me in random earlier messages. This makes it more difficult to quickly read only unread ones.

**Comment by FrayxRulez** [comment]
This issue is fixed in the latest beta
